uiplib/gui/mainGui.py

```diff
@@ -94,6 +96,7 @@ def flush(self):
                                 icon='warning')
         if ask == 'yes':
             flush_wallpapers(self.settings['pics-folder'])
+            self.update_pics_from_fav_pics()
```

Comment:
Oh sorry, I did not see that is how you were using `update_pics_from_fav_pics()`

I suggest you add it to the end of `update_pics`